docs/dev/MAINTAINERS.mdx

```diff
@@ -0,0 +1,156 @@
+# changelog.dart HACKING guide
+
+## Table of Content
+
+- Introduction
+- Code Style
+- Commit Style
+- How make the release
+
+## Introduction
+
+Welcome in the HACKING guide and how a day in a lndart.cln maintainer looks like.
+
+After this reading you should be ready to contribute to the repository and also be one of
+the next maintainer in the future if you would like!
+
+Let's begin
+
+## Code style
+
+To ensure consistency throughout the source code, keep these rules in mind as you are working:
+
+- All features or bug fixes **must be tested** by one or more specs (unit-tests).
+- All public API methods **must be documented**. (Details TBC).
+- We follow [Effective Dart: Style Guidelines][dart-style-guide].
+
+### If You Don’t Know The Right Thing, Do The Simplest Thing
+Sometimes the right way is unclear, so it’s best not to spend time on it. It’s far easier to rewrite simple code than complex code, too.
+
+### Use of `FIXME`
+
+There are two cases in which you should use a `/* FIXME: */`
+comment: one is where an optimization is possible, but it’s not clear that it’s yet worthwhile,
+and the second one is to note an ugly corner case which could be improved (and may be in a following patch).
+
+There are always compromises in code: eventually it needs to ship. `FIXME` is grep-fodder for yourself and others,
+as well as useful warning signs if we later encounter an issue in some part of the code.
+
+### Write For Today: Unused Code Is Buggy Code
+
+Don’t overdesign: complexity is a killer. If you need a fancy data structure, start with a brute force linked list. Once that’s working,
```

Comment:
```suggestion
Don’t overdesign: complexity is a killer. If you need a fancy data structure, start with a brute force linked list. One that is working,
```

Reply:
I think that `Once that’s working,` is correct?